test.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import json
+
+json_file = '7_Land_Belcher_369784.json'
+collection_file = 'collection.csv'
+
+with open(json_file) as data_file:
+    json_data = json.load(data_file)
+
+deck_number = json_data.keys()[0]
+deck_object = json_data[deck_number]
+list_of_card_groups = deck_object['list']
+
+
+def get_df_of_cards_in_type(card_type):
+    cards_df = pd.DataFrame(list_of_card_groups[card_type])
+    cards_df['card_type'] = card_type
+    cards_df_with_good_column_names = cards_df.rename(columns={'deck-col-qty': 'how_many_the_deck_needs', 'deck-col-card': 'name', 'deck-col-price': 'price'})
+    return cards_df_with_good_column_names
+
+card_type_dfs = map(get_df_of_cards_in_type, list_of_card_groups.keys())
+deck_df = pd.concat(card_type_dfs)
+deck_df['how_many_the_deck_needs'] = pd.to_numeric(deck_df['how_many_the_deck_needs'])
+deck_df['price'] = pd.to_numeric(deck_df['price'])
+deck_df['unit_price'] = deck_df['price'] / deck_df['how_many_the_deck_needs']
+
+# import list of collection, then match
+collection = pd.read_csv(collection_file, index_col=0)
+# to do: exclude basic lands?
+basic_lands = ['Forest', 'Mountain', 'Island', 'Plains', 'Swamp']
+
+def get_how_many_you_have(row):
+    return collection.loc[row['name'], 'Quantity'] if row['name'] in collection.index else 0
+
+deck_df['how_many_you_have'] = deck_df.apply(get_how_many_you_have, axis=1)
+deck_df['how_many_you_need'] = deck_df['how_many_the_deck_needs'] - deck_df['how_many_you_have']
+
+def get_cost_to_complete_playset(row):
```

Comment:
minor... technically not how much $ needed to complete "playset", rather how much $ needed to complete deck requirements